### PR TITLE
Show the full domain, not just the TLD for multipart domains

### PR DESCRIPTION
**Pull request by peternewman**
e.g. costa.co.uk rather than just .co.uk

See Fafa's here:
https://openvegemap.netlib.re/#zoom=19&lat=51.515147&lon=-0.126379

Only tested via the CI in #104 ...

### CODE REVIEW

**Comment by Rudloff** [comment]
Thanks but this was already added on the develop branch: https://github.com/Rudloff/openvegemap/commit/3299595e7e0106c264fbbea76f05d0b3c8c09bd2

I will add the tests though.

File: tests/test.js [hunk 1]
```diff
@@ -53,3 +53,17 @@ qunit.test('Popup', function (assert) {
 
     assert.equal('<ons-list-item modifier="nodivider"><div class="left list-item__title">Vegan</div> <div class="right list-item__subtitle">foo</div></ons-list-item>', popup.getPopupRows(), 'getPopupRows');
 });
+
+qunit.test('Popup website', function (assert) {
+    'use strict';
+    const popup = new Popup({'diet:vegan': 'foo', 'website': 'https://www.example.com/bar', 'contact:website': 'http://foo.com/'});
+
+    assert.equal('<ons-list-item modifier="nodivider"><div class="left list-item__title">Vegan</div> <div class="right list-item__subtitle">foo</div></ons-list-item><ons-list-item modifier="nodivider"><div class="left list-item__title">Website</div> <div class="right list-item__subtitle"><a target=" blank" rel="noopener" href="https://www.example.com/bar">example.com</a></div></ons-list-item>', popup.getPopupRows(), 'getPopupRows');
```
Comment (Rudloff) [inline tests/test.js:61]: The tests should be updated to take 339a94f14dfc7d17386ff5b498fe191e578b8264 into account (which was recently added on the develop branch):
```
>> Popup website
>> Message: getPopupRows
>> Actual: "<ons-list-item modifier=\"nodivider\"><div class=\"left list-item__title\">Vegan</div> <div class=\"right list-item__subtitle\">foo</div></ons-list-item><ons-list-item modifier=\"nodivider\"><div class=\"left list-item__title\">Website</div> <div class=\"right list-item__subtitle\"><a target=\" blank\" rel=\"noopener\" href=\"https://www.example.com/bar\">example.com</a></div></ons-list-item>"
>> Expected: "<ons-list-item modifier=\"nodivider\"><div class=\"left list-item__title\"><a target=\"_blank\" href=\"https://wiki.openstreetmap.org/wiki/Key%3Adiet:vegan\">Vegan</a></div> <div class=\"right list-item__subtitle\">foo</div></ons-list-item><ons-list-item modifier=\"nodivider\"><div class=\"left list-item__title\">Website</div> <div class=\"right list-item__subtitle\"><a target=\" blank\" rel=\"noopener\" href=\"https://www.example.com/bar\">example.com</a></div></ons-list-item>"
>> at Object.<anonymous> (file:///home/pierre/www/openvegemap/dist/test.bundle.js:23:62759)
```

File: tests/test.js [hunk 2]
```diff
@@ -53,3 +53,17 @@ qunit.test('Popup', function (assert) {
...
+});
+
+qunit.test('Popup UK website', function (assert) {
+    'use strict';
+    const popup = new Popup({'diet:vegan': 'foo', 'contact:website': 'http://www.example.co.uk/baz'});
+
+    assert.equal('<ons-list-item modifier="nodivider"><div class="left list-item__title">Vegan</div> <div class="right list-item__subtitle">foo</div></ons-list-item><ons-list-item modifier="nodivider"><div class="left list-item__title">Website</div> <div class="right list-item__subtitle"><a target=" blank" rel="noopener" href="http://www.example.co.uk/baz">example.co.uk</a></div></ons-list-item>', popup.getPopupRows(), 'getPopupRows');
```
Comment (Rudloff) [inline tests/test.js:68]: Same:
```
>> Popup UK website
>> Message: getPopupRows
>> Actual: "<ons-list-item modifier=\"nodivider\"><div class=\"left list-item__title\">Vegan</div> <div class=\"right list-item__subtitle\">foo</div></ons-list-item><ons-list-item modifier=\"nodivider\"><div class=\"left list-item__title\">Website</div> <div class=\"right list-item__subtitle\"><a target=\" blank\" rel=\"noopener\" href=\"http://www.example.co.uk/baz\">example.co.uk</a></div></ons-list-item>"
>> Expected: "<ons-list-item modifier=\"nodivider\"><div class=\"left list-item__title\"><a target=\"_blank\" href=\"https://wiki.openstreetmap.org/wiki/Key%3Adiet:vegan\">Vegan</a></div> <div class=\"right list-item__subtitle\">foo</div></ons-list-item><ons-list-item modifier=\"nodivider\"><div class=\"left list-item__title\">Website</div> <div class=\"right list-item__subtitle\"><a target=\" blank\" rel=\"noopener\" href=\"http://www.example.co.uk/baz\">example.co.uk</a></div></ons-list-item>"
>> at Object.<anonymous> (file:///home/pierre/www/openvegemap/dist/test.bundle.js:23:63311)
```

**Review by Rudloff — CHANGES_REQUESTED** [review]

File: tests/test.js (hunk 1 above)
Comment (peternewman) [reply tests/test.js]: Done

File: tests/test.js (hunk 2 above)
Comment (peternewman) [reply tests/test.js]: Done

**Comment by peternewman** [comment]
> Thanks but this was already added on the develop branch: [3299595](https://github.com/Rudloff/openvegemap/commit/3299595e7e0106c264fbbea76f05d0b3c8c09bd2)

Doh, I don't know how I missed it when searching, and funny how two of us found it within a few weeks of each other!

> I will add the tests though.

Thanks.

Will it get released fairly soon?

**Comment by peternewman** [comment]
We don't need to explicitly add psl given it's only optional on extract-domain then?
https://github.com/Rudloff/openvegemap/blob/4eb4c0c2083a97d0865fa266c38ebd29a8e067db/package.json#L23

**Comment by Rudloff** [comment]
> We don't need to explicitly add psl given it's only optional on extract-domain then?

It is still a direct dependency of extract-domain: https://www.npmjs.com/package/extract-domain
As long as it is in our `yarn.lock` file, it will be installed.

**Comment by peternewman** [comment]
Is this good too now @Rudloff ?

**Review by Rudloff — APPROVED** [review]

**Pull request merged**